plugins/modules/ntnx_acps.py

```diff
@@ -421,6 +421,10 @@ def check_acp_idempotency(old_spec, update_spec):
         if context not in update_context_list:
             return False
 
+    for context in update_context_list:
```

Comment:
Thanks for pointing it out. Btw can this be solved by comparing length of old and new context list ?

Reply:
it could possibly be solved that way, but what if the length of the list doesn't change but the contents do?   i.e.  remove one bad entry and add a new replacement one

Reply:
Above `for loop` should take care of that. It will not find that one bad entry from old_context_list in new_context_list

Reply:
And that will detect that change

Reply:
Previous for loop should take care of that. It will not find that one bad entry from old_context_list in new_context_list, which will detect the change in context lists